app/views/compute_profiles/welcome.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for(:title, _("Compute Profiles")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("list-alt", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Compute Profiles') %></h1>
+  <p><%= _("A compute profile is a way of expressing a set of defaults for VMs created on a specific compute resource
+           that can be mapped to an operator-defined label. This means an administrator can express,
+           for example, what “Small”, Medium” or “Large” means on all of the individual compute resources present for a given installation.") %></p>
```

Comment:
1. Missing quotation mark around "Medium".
2. I'd suggest replacing the Unicode quotation marks with regular `"` ones, or using the correct open/close Unicode quotation marks instead of only closing ones.